elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -339,16 +356,13 @@ def grade(element_html, data):
     weight = pl.get_integer_attrib(element, 'weight', WEIGHT_DEFAULT)
 
     submitted_key = data['submitted_answers'].get(name, None)
-    correct_key = data['correct_answers'].get(name, {'key': None}).get('key', None)
-
-    score = 0
-    if (submitted_key is not None and submitted_key == correct_key):
-        score = 1
 
     feedback = None
+    score = 0
     for option in data['params'][name]:
         if option['key'] == submitted_key:
             feedback = option.get('feedback', None)
+            score = option.get('score', SCORE_INCORRECT_DEFAULT)
```

Comment:
Questions that were generated prior to this change won't have `score` in their params, so this will give students a score of 0 even if they answered correctly. We should ensure this change is made in a backwards-compatible way. Something like this may work:

```suggestion
            score = option.get('score', SCORE_CORRECT_DEFAULT if submitted_key == correct_key else SCORE_INCORRECT_DEFAULT)
```

Reply:
Did something similar to this. Tested briefly (see screenshot in description). Should probably double check for backward compatibility one more time but should be working ok.